app/controllers/staff/organisations_controller.rb

```diff
@@ -1,23 +1,26 @@
 # frozen_string_literal: true
 
 class Staff::OrganisationsController < Staff::BaseController
-  include Secured
-
   def index
-    @organisations = Organisation.all
+    @organisations = policy_scope(Organisation)
```

Comment:
NB: You can actually write this as `@organisations = authorize policy_scope(Organisations)` but was worried about hitting Rubocop's line length rules.